acceptance/gossip_peerings_test.go

```diff
@@ -141,6 +141,8 @@ func TestGossipPeerings(t *testing.T) {
 // re-bootstrapped after a time when all nodes were down
 // simultaneously.
 func TestGossipRestart(t *testing.T) {
+	t.Skipf("TODO(pmattis): #4445")
```

Comment:
Yeah there's no good reason this should fail with 5 but not 3 nodes. Probably 5 just increases the chance that whatever is going wrong goes wrong.